pkg/resource/api.go

```diff
@@ -48,7 +48,9 @@ func NewOwnerReferenceAdder() OwnerReferenceAdder {
 type OwnerReferenceAdder struct{}
 
 func (lo OwnerReferenceAdder) Patch(cr ParentResource, list []ChildResource) ([]ChildResource, error) {
-	ref := meta.ReferenceTo(cr, cr.GroupVersionKind())
+	ref := meta.AsOwner(meta.ReferenceTo(cr, cr.GroupVersionKind()))
```

Comment:
Based on the [linked docs](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#foreground-cascading-deletion), it sounds like we need `delete` permissions on the CR in order to set the block owner deletion key. Are those permissions already configured for the shared controller? I can't remember off the top of my head

Reply:
I didn't know that, good catch! It seems we're giving all permissions on the CR instances of the CRDs that the stack deploys, so, we're good on that. See https://github.com/crossplane/crossplane/blob/35ad921/pkg/stacks/unpack.go#L440